src/com/egeniq/widget/CustomTypefaceSpan.java

```diff
@@ -0,0 +1,48 @@
+package com.egeniq.widget;
+
+import android.graphics.Paint;
+import android.graphics.Typeface;
+import android.text.TextPaint;
+import android.text.style.MetricAffectingSpan;
+
+/**
+ * Created by josvanegmond on 7/27/15.
+ * Assisting class for multiple typefaces
+ * Original code from http://stackoverflow.com/a/17961854
+ */
+public class CustomTypefaceSpan extends MetricAffectingSpan {
```

Comment:
It takes different constructor arguments. The built-in doesn't take a custom Typeface. This class has the custom Typeface as attribute, not the family string. I don't think it's parcelable then?